Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -198,20 +198,83 @@ Note: in step 5, if you choose **Use existing app registration**, make sure to d
 17. Navigate to Store, and click **Upload a custom app > Upload for ORGANIZATION-NAME**, and select the ZIP file you downloaded.
 
 
-### Grant the Demisto Bot Permissions in Microsoft Graph
+
+### In order to connect to the Azure Network Security Groups use one of the following methods:
+
+1. *Client Credentials Flow*
+2. *Authorization Code Flow*
+
+### Client Credentials Flow
+#### Grant the Demisto Bot Permissions in Microsoft Graph
 
 1. Go to your Microsoft Azure portal, and from the left navigation pane select **Azure Active Directory > App registrations**.
 2. Search for and click **Demisto Bot**.
 3. Click **API permissions > Add a permission > Microsoft Graph > Application permissions**.
-4. For the following permissions, search for,  select the checkbox and click **Add permissions**.
+4. For the following permissions, search for, select the checkbox and click **Add permissions**.
   - User.Read.All
   - Group.ReadWrite.All
   - Calls.Initiate.All
   - Calls.InitiateGroupCall.All
   - OnlineMeetings.ReadWrite.All
+  - ChannelMember.ReadWrite.All
+  - Channel.Create
+
+5. Verify that all permissions were added, and click **Grant admin consent for Demisto**.
+6. When prompted to verify granting permissions, click **Yes**, and verify that permissions were successfully added.
+
+#### Authentication Using the Client Credentials Flow
 
+1. Choose the 'Client Credentials' option in the ***Authentication Type*** parameter.
+2. Enter your Client/Application ID in the ***Bot ID*** parameter. 
+3. Enter your Client Secret in the ***Bot Password*** parameter.
+4. Save the instance.
+
+
+### Authorization Code Flow
+#### Grant the Demisto Bot Permissions in Microsoft Graph
+
+1. Go to your Microsoft Azure portal, and from the left navigation pane select **Azure Active Directory > App registrations**.
+2. Search for and click **Demisto Bot**.
+3. Click **API permissions > Add a permission > Microsoft Graph > Application permissions**.
+4. For the following permissions, search for, select the checkbox and click **Add permissions**.
+    ##### Required Application Permissions:
+      1. User.Read.All
+      2. Group.ReadWrite.All
+      3. OnlineMeetings.ReadWrite.All
+      4. ChannelMember.ReadWrite.All
+      5. Channel.Create
+
+    ##### Required Delegated Permissions:
+      1. ChannelMessage.Send
+      2. Chat.ReadWrite
+      3. ChatMessage.Send
+      4. Group.ReadWrite.All
+      5. Channel.Create
+      6. ChannelSettings.ReadWrite.All
+      7. ChatMember.ReadWrite
+      8. Chat.ReadWrite
 5. Verify that all permissions were added, and click **Grant admin consent for Demisto**.
 6. When prompted to verify granting permissions, click **Yes**, and verify that permissions were successfully added.
+7. Click **Expose an API and add Application ID URI (id only)  
```

Comment:
Is this the full name of what appears in the UI?
Need to add ** at the end of the text.
So should it be:
Click **Expose an API and add Application ID URI (id only)**.